php/commands/site.php

```diff
@@ -207,6 +207,9 @@ public function connection_mode($args, $assoc_args) {
       $action = 'set';
     }
     $env = Input::env($assoc_args, 'env');
+    if (($env == 'test' || $env == 'live') && $action == 'set') {
+      Terminus::error("Connection mode cannot be set in Test or Live environments");
```

Comment:
@ElliotFriend What do you think about moving this check down a few lines into the switch statement's 'set' case near line 222? Then we wouldn't be dup'ing the action check logic.